tot/df_utils.py

```diff
@@ -138,16 +180,8 @@ def __crossvalidation_split_df(df, k, fold_pct, fold_overlap_pct=0.0):
 
             validation data
     """
-    # Receives df with single ID column
-    assert len(df["ID"].unique()) == 1
```

Comment:
I removed this assert because `__crossvalidation_split_df` function is always called on a df grouped by `ID`